src/main.js

```diff
@@ -20,12 +20,12 @@
  *
  */
 import Vue from 'vue'
+import { translate, translatePlural } from 'nextcloud-l10n'
+
 import App from './App'
 
-Vue.prototype.t = t
-Vue.prototype.n = n
-Vue.prototype.OC = OC
-Vue.prototype.OCA = OCA
+Vue.prototype.$t = translate
```

Comment:
You use the vue-components and they need `Vue.prototype.t` to be present, otherwise they will complain. That is why e.g. in the Tasks app I have both atm. Related issue is https://github.com/nextcloud/nextcloud-vue/issues/586.

Reply:
Oooh, I did not get that! :O
We should definitely fix this!